feat: create reusable image component

disha1202 opened the pull request:
Closes: #247

disha1202 committed:
feat: moved all the .svg in src/assets/icons folder

disha1202 committed:
feat: implemented ImageSVG component

disha1202 committed:
feat: added missing semicolons

meganindya commented:
I added a new module to centralise the loading and management of assets.

It'll make it easier for you but you'll have to make changes accordingly. I've updated the description of this issue. Perhaps, close this one and create a new PR. You'll need to base on top of 984746112e32fe69f4d78c572169a99674cac708 to get visibility of the new assets module code.

disha1202 commented:
> I added a new module to centralise the loading and management of assets.
> 
> It'll make it easier for you but you'll have to make changes accordingly. I've updated the description of this issue. Perhaps, close this one and create a new PR. You'll need to base on top of [9847461](https://github.com/sugarlabs/musicblocks-v4/commit/984746112e32fe69f4d78c572169a99674cac708) to get visibility of the new assets module code.

Sure Sir, I'll close this PR and create a new one based on top of the updations made in [9847461](https://github.com/sugarlabs/musicblocks-v4/commit/984746112e32fe69f4d78c572169a99674cac708)
